config/e2e/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+namespace: kubestone-system
```

Comment:
WDYT about calling out explicitly that this is `e2e-deployment`. We could also rename `default` to `default-deployment`? Or maybe a README.md in each dirs to clarify the intent? Not sure what is best....

Reply:
I usually do not like to add -deployment, -serviceaccount, -role, ... suffixes to Kubernetes resources as the resource type explicitly specify it. This is not the case in this situation so we can go with any option. I would stick to default and e2e now but have the possibility to change it anytime.